### ISSUE TITLE
[Feature][Api] When online the workflow, if there is a timing, add an option: whether to launch a timing, and display the timing expression

### ISSUE THREAD

**Issue by xingchun-chen** [apache/dolphinscheduler#4946]
**Describe the feature**
1. When the workflow is offline, timing will be offline at the same time

2. When the workflow is online again, it is easy to forget the online timing
![image](https://user-images.githubusercontent.com/55787491/114370436-0c6ebd00-9bb2-11eb-89cb-0c6e79270c2b.png)

**Describe the solution you'd like**

When online the workflow, if there is a timing, add an option: whether to online a timing, and display the timing expression

**Comment by davidzollo** [comment]
good idea, if there are anybody who want to do this Feature, please leave a message

**Comment by Mighten** [comment]
Could you please assign this issue to me?

This work is due by *March 15, 2025*.

**Comment by Mighten** [comment]
This issue should be closed, as it has already been implemented in the latest code:

![Screenshot 2024-12-19 211141](https://github.com/user-attachments/assets/88afe952-90ec-497b-bdb3-909e470a208b)
![Screenshot 2024-12-19 211203](https://github.com/user-attachments/assets/8b03fa1d-4a69-405a-be36-b1fb79fa70e8)
![Screenshot 2024-12-19 211242](https://github.com/user-attachments/assets/11a4524e-60bd-4298-88e1-b96b15a98cf6)
![Screenshot 2024-12-19 211316](https://github.com/user-attachments/assets/fe542531-a8f2-4c6e-8c4b-cb0911608931)
![Screenshot 2024-12-19 211325](https://github.com/user-attachments/assets/03edb898-ebcf-4903-aad4-fd7764e46d2d)

@davidzollo
